### PR TITLE
Consolidate deployer lsig functions

### PR DESCRIPTION
**Pull request by ratik21**
- rename `loadLsig` to `loadLogic`
- merge `loadBinaryMultisig(..)` into `loadMultisig()`
- update readme

### CODE REVIEW

File: examples/multisig/README.md [hunk 1]
```diff
@@ -35,15 +35,15 @@ Sign again with addr2 (as threshold is set to 2)
 goal clerk multisig signprogram -L <path>/signed.msig -a addr2 -A <multisig_hash> -d ~/<data_directory>
 ```
 
-The logic sig file generated using the commands above is a binary file. To use it, you have to store it in the `/assets` directory. You can also decompile it to a readable format (command provided below) and save it to `/assets`.
+The logic sig file generated using the commands above is a binary file. To use it, you have to store it in the `/assets` directory with `.bmsig` extension. You can also decompile it to a readable format (command provided below) and save it to `/assets` with `.msig` extension.
```
Comment (robert-zaremba) [inline examples/multisig/README.md:38]:
```suggestion
The logic sig file generated using the commands above is a binary file. To use it, you have to store it in the `/assets` directory with `.bmsig` extension (binary encoded LogicSig). You can also decompile it to a readable format (command provided below) and save it to `/assets` with `.msig` extension.
```

**Review by robert-zaremba — COMMENTED** [review]
I have one more suggestion: the content of the msig / bmsig file is a serialized LogicSig object. How about changing the extension of the files form msig -> lsig? (we will need to do it in examples as well, so if you like the idea, merge the latest master into this branch because the directories in examples have changed)

**Comment by ratik21** [comment]
> I have one more suggestion: the content of the msig / bmsig file is a serialized LogicSig object. How about changing the extension of the files form msig -> lsig? (we will need to do it in examples as well, so if you like the idea, merge the latest master into this branch because the directories in examples have changed)

Yes, i think that will be good and more clear to the user. :+1:

**Review by robert-zaremba — APPROVED** [review]
good job

**Pull request merged**